### DISCUSSION TITLE
What is equivalent code for parent.$.fancybox.close(); in latest version 4?

### DISCUSSION THREAD

**Discussion by skmusharaf786** [fancyapps/ui#265] [Q&A]
We are using fancybox with Jquery in our application. 

Could you please help us with equivalent code for parent.$.fancybox.close() in latest version 4 of fancybox?

Thankyou

**Comment by fancyapps** [comment]
Hi,

Fancybox before the fourth version was jQuery plugin and was located inside jQuery object (`$` is alias of `jQuery`).

Starting from v4, Fancybox is ES6 class. If you are using UMD build then it is available globally as `Fancybox` object and you can use API like this:

```js
parent.Fancybox.close();
```

Demo - https://codesandbox.io/s/fancybox-iframe-k4i41?file=/iframe-1.html

**Reply by skmusharaf786** [reply]
Thankyou 👍
